packages/expo-file-system/android/src/main/java/expo/modules/filesystem/next/FileSystemDirectory.kt

```diff
@@ -42,9 +42,10 @@ class FileSystemDirectory(file: File) : FileSystemPath(file) {
     validateType()
     validatePermission(Permission.READ)
     return file.listFiles()?.map {
+      val uriString = Uri.fromFile(it).toString()
       mapOf(
         "isDirectory" to it.isDirectory,
-        "path" to it.path
+        "uri" to if (uriString.endsWith("/")) uriString else "$uriString/"
```

Comment:
I'm not sure about adding a trailing slash to a non directory resources. It might be a bit confusing

Reply:
Hmm, it's internal only, we clean it up when creating the actual file/directory instances.